csspositiong2.css

```diff
@@ -0,0 +1,51 @@
+.parent {
```

Comment:
Era ok sa pui elementele comune care sunt legate de child in clasa .child{} si cele specifice in .child1{},...